src/assets/custom/css/custom-softmod-feature.scss

```diff
@@ -0,0 +1,751 @@
+    }
+
+    .g-pt-50--desk {
+        padding-top: 50px!important;
+    }
+
+    .g-pt-100--desk {
+        padding-top: 100px!important;
+    }
+
+    .g-pb-25--desk {
+        padding-bottom: 25px!important;
+    }
+
+    .g-pb-50--desk {
+        padding-bottom: 50px!important;
+    }
+
+    .g-pb-100--desk {
+        padding-bottom: 100px!important;
+    }
+
+    .g-pb-200--desk {
+        padding-bottom: 200px!important;
+    }
+
+}
+
+/********************************
+    Helpers - Backgrounds
+********************************/
+
+.bg-slate {
+    background-color: #2B3342;
+}
+
+.bg-blue {
+    background-color:#0F4071!important;
+}
+
+.bg-white {
+    background-color: #FFFFFF;
+}
+
+.bg-coral {
+    background-color: #FB7268;
+}
+
+.bg-slate--pattern {
+    background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-slate-pattern-2.jpg');
+    background-size: 100%;
+    background-repeat: no-repeat;
+}
+
+.bg-testimonials {
+    background:url('{{site.baseurl}}/assets/custom/img/softmod/bg-testimonials.png');
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+.bg-compass-cta {
+    background:url('{{site.baseurl}}/assets/custom/img/softmod/bg-compass-cta.jpg');
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+@media (min-width: 800px) {
+
+    .bg-sustainable-change {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-sustainable-change.svg');
+        background-position: center bottom 40px;
+        background-repeat: no-repeat;
+        background-size: 55% auto;
+    }
+
+    .bg-innovation {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-innovation.svg');
+        background-position: center bottom 40px;
+        background-repeat: no-repeat;
+        background-size: 45% auto;
+    }
+
+    .bg-technology {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-technology.svg');
+        background-position: center bottom 30px;
+        background-repeat: no-repeat;
+        background-size: 70% auto;
+    }
+
+    .bg-business-alignment {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-business-alignment.svg');
+        background-position: center bottom 40px;
+        background-repeat: no-repeat;
+        background-size: 50% auto;
+    }
+
+    .bg-people-culture {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-people-culture.svg');
+        background-position: center bottom 50px;
+        background-repeat: no-repeat;
+        background-size: 60% auto;
+    }
+
+    .bg-risk-management {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-risk-management.svg');
+        background-position: center bottom 50px;
+        background-repeat: no-repeat;
+        background-size: 50% auto;
+    }
+
+    .bg-testimonials {
+        background:url('{{site.baseurl}}/assets/custom/img/softmod/bg-testimonials.jpg');
```

Comment:
Looks like the shorthand `background` is being used along side longhand attributes like `background-size` etc